src/types/index.ts

```diff
@@ -57,6 +57,17 @@ export type BookmarkInput = {
   cmsId?: string
 }
 
+export type RemovedBookmark = {
+  _id: string
+}
+
+export type AddBookmarkInput = {
+  url: string
+  collectionId: string
+  userId: string
+  label?: string
+  cmsId?: string
+}
```

Comment:
For type definitions that are only relevant to a narrow scope of file(s) like these, I think it is fine to define them in the relevant file, and use `src/types` for more global types that are imported across different concerns. It's very easy to end up with a ton of type definitions for various things that get confused